src/main/scala/com/lifeomic/fhirlib/v3/datatypes/Reference.scala

```diff
@@ -8,6 +8,11 @@ class Reference(val reference: Option[String],
       val uri = new java.net.URI(reference.get)
       if (uri.getFragment != null) {
         return Some(uri.getFragment)
+      } else if (uri.getScheme == "urn" && uri.getSchemeSpecificPart.split(":")(0) == "uuid") {
+        val parts = uri.getSchemeSpecificPart.split(":");
+        if (parts(0) == "uuid") {
```

Comment:
Don't need this check, already done in the if statement

Reply:
oops, meant to remove that